tests/writable-areas/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Check that snap apps and services can write to writable areas.
+prepare: |
+  snapbuild ../lib/snaps/data-writer .
+restore: |
+  rm data-writer_1.0_all.snap
+execute: |
+  snap install data-writer_1.0_all.snap
+
+  echo "Apps can write to writable areas"
+  data-writer.app
+  [ -f /var/snap/data-writer/x1/from-app ] || exit 1
+  [ -f /var/snap/data-writer/common/from-app ] || exit 1
+  [ -f /root/snap/data-writer/x1/from-app ] || exit 1
+  # TODO: As soon as `snap run` is used (which creates this directory),
```

Comment:
Yeah but then it's not quite testing as much as it should. And the service will still be missing it, so might as well keep them consistent. What do you think?